src/manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<updates>
+    <!-- update sample from Joomla at https://docs.joomla.org/Deploying_an_Update_Server
+    <update>
+        <name>Weblinks Extension Package</name>
+        <description>Joomla! CMS Weblinks Package</description>
+        <element>pkg_weblinks</element>
+        <type>package</type>
+        <version>3.6.0</version>
+        <client>site</client>
```

Comment:
we should make sure that the plugin has enough permissions as a `site`. If not this should be changed to `administrator` or `1`.
Check here for more info:
https://docs.joomla.org/Deploying_an_Update_Server